### PR TITLE
Read mappings from sierra debug info

### PR DESCRIPTION
**Pull request by piotmag769**
Related to #71

## Introduced changes

- read statements functions map from sierra debug info

### CODE REVIEW

File: crates/cairo-profiler/src/sierra_loader.rs [hunk 1]
```diff
@@ -125,6 +145,30 @@ pub fn compile_sierra_and_add_compiled_artifacts_to_cache(
...
+            );
+            annotations["statements_functions"].take()
+        })
+        .map(|statements_functions| {
+            let map =
+                serde_json::from_value::<HashMap<StatementIdx, Vec<String>>>(statements_functions)
+                    .expect("Wrong statements function map format");
+            StatementsFunctionsMap(map)
+        })
+}
```
Comment (MaksymilianDemitraszek) [inline crates/cairo-profiler/src/sierra_loader.rs:170]: I don't know when its Some and when it's none. Please create a sub function which returns the data always and in `maybe_get_statements_functions_map` store only the `maybe` logic

File: crates/cairo-profiler/src/sierra_loader.rs [hunk 2]
```diff
@@ -33,6 +35,17 @@ impl SierraProgram {
     }
 }
 
+#[allow(dead_code)]
+#[derive(Default, Clone)]
+pub struct StatementsFunctionsMap(HashMap<StatementIdx, Vec<String>>);
```
Comment (MaksymilianDemitraszek) [inline crates/cairo-profiler/src/sierra_loader.rs:40]: What is `Vec<String>`?

**Review by MaksymilianDemitraszek — CHANGES_REQUESTED** [review]

File: crates/cairo-profiler/src/sierra_loader.rs (hunk 2 above)
Comment (piotmag769) [reply crates/cairo-profiler/src/sierra_loader.rs:40]: Will change to `FunctionName`, good idea

File: crates/cairo-profiler/src/sierra_loader.rs (hunk 1 above)
Comment (piotmag769) [reply crates/cairo-profiler/src/sierra_loader.rs]: Done

**Review by MaksymilianDemitraszek — APPROVED** [review]

**Pull request merged**